crypto/default/src/main/java/org/keycloak/crypto/def/BCOCSPProvider.java

```diff
@@ -436,7 +437,7 @@ protected List<String> getResponderURIs(X509Certificate cert) throws Certificate
                     if (ad.getAccessMethod().equals(AccessDescription.id_ad_ocsp)) {
                         // See https://www.ietf.org/rfc/rfc2560.txt, 3.1 Certificate Content
                         if (ad.getAccessLocation().getTagNo() == GeneralName.uniformResourceIdentifier) {
-                            DERIA5String value = DERIA5String.getInstance(ad.getAccessLocation().getName());
+                            ASN1IA5String value = DERIA5String.getInstance(ad.getAccessLocation().getName());
```

Comment:
Minor: I think `org.bouncycastle.asn1.DERIA5String` can be removed from imports in this class?

Reply:
It cannot, it's still used by DERIA5String.getInstance